### ISSUE TITLE
CQADupstackProgrammersRetrieval results are missing for a lot of models

### ISSUE THREAD

**Issue by x-tabdeveloping** [embeddings-benchmark/results#63]
A lot of models in the `MTEB(eng, classic)` tab are missing CQADupstackProgrammersRetrieval results and therefore get nan mean, and don't get displayed in the plot.
I checked and the files are missing here, in the results repo. Is this something we should run (@Muennighoff) or something we can fetch from the old results (@Samoed )?

**Comment by Samoed** [comment]
I think this should be runned, because curently when creating model card, then all CQADupstack datasets are combined
https://github.com/embeddings-benchmark/mteb/blob/a6ce6f9b7050c1fad60e0c6e8985afa9356e2728/mteb/create_meta.py#L104

**Comment by KennethEnevoldsen** [comment]
Yep. I actually also think this will leave to a difference in the mean (which is a bit annoying). A solution is to allow custom aggregators for some benchmarks (might also fix the issue with displaying bright)

**Comment by KennethEnevoldsen** [comment]
This have now been resolved so will close this
